src/zoid/qr-code/component.jsx

```diff
@@ -143,6 +147,12 @@ export function QRCodeContainer({
         frame.classList.remove(CLASS.INVISIBLE);
         frame.classList.add(CLASS.VISIBLE);
 
+        // Close is now controlled by QRCard on smart-payment-buttons
+        const closeButton = document.querySelector('#close');
```

Comment:
Just remove this.

Reply:
@mnicpt 
I think that make sense if we decide to entirely remove the close button from checkout-components.
I replied the bellow comment from Daniel, explain why maybe we should remove it only after the child iframe is loaded.
If we decide to move forward with that, I will be happy to remove the close button on this PR.

Reply:
@andregardi can you remove this line? Based on the convo, it seems like prerender is handlnig it.